content/en/docs/concepts/architecture/nodes.md

```diff
@@ -291,6 +291,13 @@ includes all containers started by the kubelet, but not containers started direc
 If you want to explicitly reserve resources for non-Pod processes, follow this tutorial to
 [reserve resources for system daemons](/docs/tasks/administer-cluster/reserve-compute-resources/#system-reserved).
 
+## Node topology
+
+{{< feature-state state="alpha" >}}
+
+If you have enabled the `TopologyManager`
+[feature gate](/docs/reference/command-line-tools-reference/feature-gates/) then
+the kubelet can pay attention to topology hints.
```

Comment:
Maybe something like "the kubelet can use topology hints to when making resource assignment decisions"